docs/content/how-to/disconnected/known-issues.md

```diff
@@ -9,4 +9,38 @@ When you work in a disconnected environment the OLM catalog sources will be stil
 
 The most practical one is the first choice. To proceed with this option, you will need to follow [these instructions](https://docs.openshift.com/container-platform/4.14/installing/disconnected_install/installing-mirroring-disconnected.html). The process will make sure all the images get mirrored and also the ICSP will be generated properly.
 
-Additionally when you're provisioning the HostedCluster you will need to add a flag to indicate that the OLMCatalogPlacement is set to `Guest` because if that's not set, you will not be able to disable them.
\ No newline at end of file
+Additionally when you're provisioning the HostedCluster you will need to add a flag to indicate that the OLMCatalogPlacement is set to `Guest` because if that's not set, you will not be able to disable them.
+
+## Hypershift operator is failing to reconcile in Disconnected environments
+
+If you are operating in a disconnected environment and have deployed the Hypershift operator, you may encounter an issue with the UWM telemetry writer. Essentially, it exposes Openshift deployment data in your RedHat account, but this functionality does not operate in a disconnected environments.
+
+**Symthoms:**
+
+- The Hypershift operator looks running fine in the `hypershift` namespace but even if you creates the Hosted Cluster nothing happens.
```

Comment:
s/The Hypershift operator looks running fine/The Hypershift operator appears to be running correctly